libcudacxx/test/libcudacxx/std/utilities/template.bitset/bitset.members/to_ullong.pass.cpp

```diff
@@ -67,7 +67,7 @@ __host__ __device__ TEST_CONSTEXPR_CXX14 bool test()
 int main(int, char**)
 {
   test();
-#if TEST_STD_VER >= 2014 && (_CCCL_CUDACC_AT_LEAST(11, 8) || !_CCCL_COMPILER(MSVC))
+#if !_CCCL_COMPILER(MSVC)
```

Comment:
This condition should be always true now, since the old one is:
```
#if true && (true || !_CCCL_COMPILER(MSVC))
```
And or-ing with `true` is always true. I think we could remove this check entirely.

Applies a few more times below.